packages/office-ui-fabric-react/src/components/Nav/Nav.scss

```diff
@@ -103,13 +103,16 @@ $linkRightPadding: 20px;
   display: block;
   position: relative;
 
+  & button {
```

Comment:
This is way too generic. Why can't we target a specific class?

Think of it this way; if a nav element nests multiple buttons, what happens?

Reply:
We definitely can now that every Nav entry is a button